<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1338</width>
    <height>856</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(245, 250, 254);
background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="Icon_Widget" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget{
	background-color: rgb(31, 156, 233);
}
QPushButton{
	color:white;
	height:50px;
	border:none;
}

QPushButton:Checked{
	background-color:#f5fafe;
	color:#1f95ef;
	font-weight:bold;
	border-radius:10px;
}


</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>65</width>
             <height>80</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="C:/Users/Strix/Desktop/SideMenu/Resources/resources.qrc">:/images/profile 2white.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>13</number>
         </property>
         <item>
          <widget class="QPushButton" name="Dashboard_S">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="C:/Users/Strix/Desktop/SideMenu/Resources/resources.qrc">
             <normaloff>:/images/dashboard white.png</normaloff>
             <normalon>:/images/dashboard Blue.png</normalon>:/images/dashboard white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Profile_S">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="C:/Users/Strix/Desktop/SideMenu/Resources/resources.qrc">
             <normaloff>:/images/user (white).png</normaloff>
             <normalon>:/images/user (blue).png</normalon>:/images/user (white).png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Messages_S">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="C:/Users/Strix/Desktop/SideMenu/Resources/resources.qrc">
             <normaloff>:/images/comments white.png</normaloff>
             <normalon>:/images/comments.png</normalon>:/images/comments white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Notifications_S">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="C:/Users/Strix/Desktop/SideMenu/Resources/resources.qrc">
             <normaloff>:/images/Notification white.png</normaloff>
             <normalon>:/images/Notification Blue.png</normalon>:/images/Notification white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Settings_S">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="C:/Users/Strix/Desktop/SideMenu/Resources/resources.qrc">
             <normaloff>:/images/Setting white.png</normaloff>
             <normalon>:/images/Setting Blue.png</normalon>:/images/Setting white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>548</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="SignOut_S">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="C:/Users/Strix/Desktop/SideMenu/Resources/resources.qrc">
             <normaloff>:/images/power white.png</normaloff>:/images/power white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="Icon_Name_Widget" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget{
	background-color: rgb(31, 156, 233);
}
QPushButton{
	color:white;
	text-align:left;
	height:30px;
	border:None;
	padding:8px;
	border-top-left-radius:10px;
	border-bottom-left-radius:10px;
}
QPushButton:Checked{
	background-color:#f5fafe;
	color:#1f95ef;
	font-weight:bold;
}




</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>75</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>75</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="C:/Users/Strix/Desktop/SideMenu/Resources/resources.qrc">:/images/profile 2white.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="margin">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Bahnschrift SemiBold Condensed</family>
             <pointsize>15</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:White</string>
           </property>
           <property name="text">
            <string>Side Bar</string>
           </property>
           <property name="margin">
            <number>4</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>15</number>
         </property>
         <item>
          <widget class="QPushButton" name="Dashboard_B">
           <property name="text">
            <string>Dashboard</string>
           </property>
           <property name="icon">
            <iconset resource="C:/Users/Strix/Desktop/SideMenu/Resources/resources.qrc">
             <normaloff>:/images/dashboard white.png</normaloff>
             <normalon>:/images/dashboard Blue.png</normalon>:/images/dashboard white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Profile_B">
           <property name="text">
            <string>Profile</string>
           </property>
           <property name="icon">
            <iconset resource="C:/Users/Strix/Desktop/SideMenu/Resources/resources.qrc">
             <normaloff>:/images/user (white).png</normaloff>
             <normalon>:/images/user (blue).png</normalon>:/images/user (white).png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Messages_B">
           <property name="text">
            <string>Messages</string>
           </property>
           <property name="icon">
            <iconset resource="C:/Users/Strix/Desktop/SideMenu/Resources/resources.qrc">
             <normaloff>:/images/comments white.png</normaloff>
             <normalon>:/images/comments.png</normalon>:/images/comments white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Notifications_B">
           <property name="text">
            <string>Notifications</string>
           </property>
           <property name="icon">
            <iconset resource="C:/Users/Strix/Desktop/SideMenu/Resources/resources.qrc">
             <normaloff>:/images/Notification white.png</normaloff>
             <normalon>:/images/Notification Blue.png</normalon>:/images/Notification white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Settings_B">
           <property name="text">
            <string>Settings</string>
           </property>
           <property name="icon">
            <iconset resource="C:/Users/Strix/Desktop/SideMenu/Resources/resources.qrc">
             <normaloff>:/images/Setting white.png</normaloff>
             <normalon>:/images/Setting Blue.png</normalon>:/images/Setting white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>548</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="SignOut_B">
           <property name="text">
            <string>SignOut</string>
           </property>
           <property name="icon">
            <iconset resource="C:/Users/Strix/Desktop/SideMenu/Resources/resources.qrc">
             <normaloff>:/images/power white.png</normaloff>
             <normalon>:/images/power blue.png</normalon>:/images/power white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QWidget" name="widget_3" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QWidget" name="Header" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="Menu">
              <property name="styleSheet">
               <string notr="true">border:none;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="C:/Users/Strix/Desktop/SideMenu/Resources/resources.qrc">
                <normaloff>:/images/menu-bar blue.png</normaloff>:/images/menu-bar blue.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>15</number>
            </property>
            <item>
             <widget class="QPushButton" name="Search">
              <property name="styleSheet">
               <string notr="true">border:none;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="C:/Users/Strix/Desktop/SideMenu/Resources/resources.qrc">
                <normaloff>:/images/Search Icon Blue.png</normaloff>:/images/Search Icon Blue.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>100</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Profile">
              <property name="styleSheet">
               <string notr="true">border:none;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="C:/Users/Strix/Desktop/SideMenu/Resources/resources.qrc">
                <normaloff>:/images/user (blue).png</normaloff>:/images/user (blue).png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>30</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0.567, y2:0.590545, stop:0 rgba(31, 156, 233, 255), stop:1 rgba(255, 255, 255, 255));
background-color: rgb(255, 255, 255);
</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="Dashboard_Page">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <pointsize>25</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0.567, y2:0.590545, stop:0 rgba(31, 156, 233, 255), stop:1 rgba(255, 255, 255, 255));
</string>
             </property>
             <property name="text">
              <string>Dashboard</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Profile_Page">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <pointsize>25</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0.567, y2:0.590545, stop:0 rgba(31, 156, 233, 255), stop:1 rgba(255, 255, 255, 255));
</string>
             </property>
             <property name="text">
              <string>Profile</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Messages_Page">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <pointsize>25</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0.567, y2:0.590545, stop:0 rgba(31, 156, 233, 255), stop:1 rgba(255, 255, 255, 255));
</string>
             </property>
             <property name="text">
              <string>Messages</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Notifications_Page">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <pointsize>25</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0.567, y2:0.590545, stop:0 rgba(31, 156, 233, 255), stop:1 rgba(255, 255, 255, 255));
</string>
             </property>
             <property name="text">
              <string>Notifications</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Settings_Page">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="font">
              <font>
               <pointsize>25</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0.567, y2:0.590545, stop:0 rgba(31, 156, 233, 255), stop:1 rgba(255, 255, 255, 255));
</string>
             </property>
             <property name="text">
              <string>Settings</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="C:/Users/Strix/Desktop/SideMenu/Resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Menu</sender>
   <signal>toggled(bool)</signal>
   <receiver>Icon_Widget</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>468</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Menu</sender>
   <signal>toggled(bool)</signal>
   <receiver>Icon_Name_Widget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>524</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Settings_S</sender>
   <signal>toggled(bool)</signal>
   <receiver>Settings_B</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Notifications_S</sender>
   <signal>toggled(bool)</signal>
   <receiver>Notifications_B</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Messages_S</sender>
   <signal>toggled(bool)</signal>
   <receiver>Messages_B</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Profile_S</sender>
   <signal>toggled(bool)</signal>
   <receiver>Profile_B</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Dashboard_S</sender>
   <signal>toggled(bool)</signal>
   <receiver>Dashboard_B</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SignOut_S</sender>
   <signal>toggled(bool)</signal>
   <receiver>SignOut_B</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>824</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>827</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Dashboard_B</sender>
   <signal>toggled(bool)</signal>
   <receiver>Dashboard_S</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>60</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Profile_B</sender>
   <signal>toggled(bool)</signal>
   <receiver>Profile_S</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Messages_B</sender>
   <signal>toggled(bool)</signal>
   <receiver>Messages_S</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Notifications_B</sender>
   <signal>toggled(bool)</signal>
   <receiver>Notifications_S</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Settings_B</sender>
   <signal>toggled(bool)</signal>
   <receiver>Settings_S</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SignOut_B</sender>
   <signal>toggled(bool)</signal>
   <receiver>SignOut_S</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>801</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>795</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SignOut_B</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>817</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>899</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SignOut_S</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>841</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>901</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
